src/logion/model/locrequest.model.ts

```diff
@@ -344,7 +344,10 @@ export class LocMetadataItem extends Child {
     @Column({ length: 255 })
     name?: string;
 
-    @Column({ length: 255 })
+    @Column({ name: "value",  length: 255, nullable: true})
```

Comment:
Given that it's only a "cache" of published info, I don't really see the need for this extra precaution.

Reply:
If we need to rollback the deployed app (i.e. redeploy the previous version), it is important that the altered schema remains backward compatible, otherwise the previous version of the app may not start.